src/components/misc/TargetImage.tsx

```diff
@@ -1,22 +1,20 @@
-import { MutableRefObject } from 'react';
+import { HTMLProps, MutableRefObject } from 'react';
 
-import { useTargetRef } from '../../src';
+import { useTargetRef } from '../../hooks';
 
-export function Image({
-  src,
-  style,
-}: {
+interface TargetImageProps extends Omit<HTMLProps<HTMLImageElement>, 'src'> {
   src: string;
-  style?: React.CSSProperties;
-}) {
+}
+
+export function TargetImage(props: TargetImageProps) {
+  const { src, style, ...otherProps } = props;
   const ref = useTargetRef() as MutableRefObject<HTMLImageElement>;
 
   return (
     <img
-      alt="story"
-      id="story-image"
```

Comment:
You must have broken something by removing the `id`. See `utils/CommitedRoisButton.tsx`